src/containers/Review/ReviewPageWrapper.tsx

```diff
@@ -97,7 +97,7 @@ const ReviewPageWrapper: React.FC = () => {
 
       <Form>
         {reviewSections.map((reviewSection) => {
-          const { id, firstName, lastName } = currentUser as User
+          const { userId: id, firstName, lastName } = currentUser as User
           const assignedToYou = reviewSection.assigned?.id === id
```

Comment:
```suggestion
          const { userId, firstName, lastName } = currentUser as User
          const assignedToYou = reviewSection.assigned?.id === userId
```

Reply:
Yes, good catch -- this is what it does sometimes when you use the "Rename Symbol" feature in VSCode. Must have missed this one.